### ISSUE TITLE
[Feature Request] Coin Trend page should be from the exchange NOT Coin Market Cap

### ISSUE THREAD

**Issue by NigglesSquiggles** [bTayFla/PtTracker#140]
In the tools section, under Coin Trend. Please make the data from the exchange so the information is relevant. If I need coin market cap info, I'll just go there.

**Comment by riggermortum** [comment]
+1 Thought the same yesterday when I was looking at other coins to add to my trading pairs, then realizing a lot of them dont exist on the exchange im using.

**Comment by LiamAird** [comment]
![binance](https://user-images.githubusercontent.com/11542070/41300667-97f73c70-6e66-11e8-928e-9ace2fd31f90.PNG)

Will this information be what you are looking for, for Binance coin trends?

**Comment by NigglesSquiggles** [comment]
Yes that’s correct. Maybe add an option to see other coins on other exchanges? But default should be the exchange your bot is on.

**Comment by LiamAird** [comment]
Will add all supported exchanges + coinmarketcap.
